pagopa/io-app-design-system
# [IOAPPX-301] Add the new `satispay` payment logo (both small and big)

## Short description
This PR adds the new **Satispay** payment logo (both small and big)

## List of changes proposed in this pull request
- Add the new components to `LogoPayment` and `LogoPaymentExt`
- Add the original SVG files in the `originals` folder

### Preview
<img width="332" alt="Screenshot 2024-05-22 at 10 30 17" src="https://github.com/pagopa/io-app-design-system/assets/1255491/7b962541-1af3-4f4a-87d5-fa5ca39ecc09">


## How to test
1. Launch the example app
2. Check out the **Logos** page
